core-services/_artifacts.tf

```diff
@@ -18,7 +18,9 @@ locals {
     version          = data.azurerm_kubernetes_cluster.cluster.kubernetes_version
     platform_version = ""
   }
-
+  specs_azure = {
```

Comment:
You need to add this block to the artifact under the `specs` block under the key `azure`